CHANGELOG.md

```diff
@@ -113,6 +113,7 @@
 * Node: Added DBSize command ([#1932](https://github.com/valkey-io/valkey-glide/pull/1932))
 * Node: Added GeoAdd command ([#1980](https://github.com/valkey-io/valkey-glide/pull/1980))
 * Node: Added ZRevRank command ([#1977](https://github.com/valkey-io/valkey-glide/pull/1977))
+* Node: Added GeoUnit command ([#1988](https://github.com/valkey-io/valkey-glide/pull/1988))
```

Comment:
geodist

Reply:
Wow, how did that happen, prolly mispaste